build/bazel/remote/execution/v0.2/remote_execution.proto

```diff
@@ -994,6 +1009,78 @@ message GetTreeResponse {
   string next_page_token = 2;
 }
 
+// A request message for
+// [Capabilities.GetCapabilities][google.devtools.remoteexecution.v1test.Capabilities.GetCapabilities].
+message GetCapabilitiesRequest {
+  // The instance of the execution system to operate against. A server may
+  // support multiple instances of the execution system (with their own workers,
+  // storage, caches, etc.). The server MAY require use of this field to select
+  // between them in an implementation-defined fashion, otherwise it can be
+  // omitted.
+  string instance_name = 1;
+}
+
+// A response message for
+// [Capabilities.GetCapabilities][google.devtools.remoteexecution.v1test.Capabilities.GetCapabilities].
+message GetCapabilitiesResponse {
+  // Capabilities of the remote cache system.
+  CacheCapabilities cache_capabilities = 1;
+
+  // Capabilities of the remote execution system.
+  ExecutionCapabilities execution_capabilities = 2;
+}
+
+// The digest function used for converting values into keys for CAS and Action
+// Cache.
+enum DigestFunction {
+  UNKNOWN = 0;
+  SHA256 = 1;
+  SHA1 = 2;
+  MD5 = 3;
+}
+
+// Describes the server/instance capabilities for updating the action cache.
+message ActionCacheUpdateCapabilities {
+  bool update_enabled = 1;
+}
+
+// Allowed values for priority in
+// [ResultsCachePolicy][google.devtools.remoteexecution.v1test.ResultsCachePolicy]
+// Used for querying both cache and execution valid priority ranges.
+message PriorityCapabilities {
```

Comment:
These are related to #11 and #12; and it strikes me as possibly problematic that they use inverse conventions for priotity (for the cache, higher priority is stronger, but for execution, lower priority is stronger).

Reply:
Oh, that was a mistake! I have now inverted the priority for cache to make lower=better for both for consistency.